lib/babel.js

```diff
@@ -2,6 +2,8 @@
 var resolveFrom = require('resolve-from');
 var createEspowerPlugin = require('babel-plugin-espower/create');
 var requireFromString = require('require-from-string');
+var hasha = require('hasha');
+var cache = require('cacha')('.ava/cache');
```

Comment:
@sindresorhus done.

@jamestalmage you are perfectly right. Until #202 is resolved we can hash with `ava` version.